### DISCUSSION TITLE
OC performance tips and tricks documentation

### DISCUSSION THREAD

**Discussion by scleaver** [OrchardCMS/OrchardCore#9298] [Ideas]
Hello,

I would like to start some documentation on tips and tricks on making your OC install go like a rocket. Do you think this would be a valuable part of the documentation?

I am going to need some pointers on this though from the team so I can collate them. 

For instance, one thing that I see affecting performance scores is "initial server response time"... not sure how to address this.

Anyway I would be happy to start some documentation on this if others think it would be valuable.

**Comment by Piedone** [comment]
Sounds like a great idea! By "initial server response time" do you mean the time to the first response when launching the app or for any request? If the former, then compiling with ReadyToRun can help a lot (and not having any slow custom code that runs during shell start). If the latter then that's a general performance question about whatever you do on the server-side and I don't think we can have non-generic advice there (like use a fast and low-latency DB, profile your code). This may also be quite related to what happens on the network beyond the app's reach (so e.g. having fast DNS resolution, CDN, using HTTP/2 or 3 can help).
